src/Routes.js

```diff
@@ -68,7 +68,7 @@ const Routes = (props) => (
     <Route path="/nih_pilot_info" component={NIHPilotInfo} />
     <AuthenticatedRoute path="/admin_console" component={AdminConsole} props={props} rolesAllowed={[USER_ROLES.admin]} />
     <AuthenticatedRoute path="/admin_manage_users" component={AdminManageUsers} props={props} rolesAllowed={[USER_ROLES.admin]} />
-    <AuthenticatedRoute path="/admin_manage_dac" component={AdminManageDac} props={props} rolesAllowed={[USER_ROLES.admin]} />
+    <AuthenticatedRoute path="/manage_dac/:role?" component={ManageDac} props={props} rolesAllowed={[USER_ROLES.admin, USER_ROLES.chairperson]} />
```

Comment:
I don't think you need to pass a role query param here. We know the user's roles from the stored user in session.